packages/webview_flutter_lwe/tizen/src/webview.h

```diff
@@ -50,16 +54,14 @@ class WebView : public PlatformView {
                                                        size_t height);
 
  private:
-  void HandleMethodCall(
-      const flutter::MethodCall<flutter::EncodableValue>& method_call,
-      std::unique_ptr<flutter::MethodResult<flutter::EncodableValue>> result);
-  void HandleCookieMethodCall(
-      const flutter::MethodCall<flutter::EncodableValue>& method_call,
-      std::unique_ptr<flutter::MethodResult<flutter::EncodableValue>> result);
-
-  void ApplySettings(const flutter::EncodableMap& settings);
+  void HandleTizenWebViewMethodCall(const FlMethodCall& method_call,
+                                    std::unique_ptr<FlMethodResult> result);
+  void HandleCookieMethodCall(const FlMethodCall& method_call,
+                              std::unique_ptr<FlMethodResult> result);
```

Comment:
Do you plan to update webview_flutter_tizen as well for consistency? I think you can do it in this PR.

Reply:
I updated